docs/matlab/matlab.rst

```diff
@@ -0,0 +1,23 @@
+MATLAB® Tutorial
```

Comment:
I'm assuming that someone who will use this is not familiar with the Python development environment, therefore I think we should point out some resource for installing a virtual env and running pip install rocketpy, mainly for a windows user

Reply:
> We start by configuring the Python environment. You can do so by following the guidelines presented in the MATLAB® documentation: [Configure Your System to Use Python - MATLAB & Simulink (mathworks.com)](https://www.mathworks.com/help/matlab/matlab_external/install-supported-python-implementation.html?searchHighlight=python&s_tid=srchtitle_python_4).

We currently have this link with a quick tutorial from Mathworks them selves. What do you think?